Tools/WebKitTestRunner/ios/UIScriptControllerIOS.mm

```diff
@@ -1356,9 +1359,19 @@ static UIDeviceOrientation toUIDeviceOrientation(DeviceOrientation orientation)
 void UIScriptControllerIOS::toggleCapsLock(JSValueRef callback)
 {
     m_capsLockOn = !m_capsLockOn;
-    auto *keyboardEvent = createUIPhysicalKeyboardEvent(@"capsLock", [NSString string], m_capsLockOn ? UIKeyModifierAlphaShift : 0,
-        kUIKeyboardInputModifierFlagsChanged, m_capsLockOn);
-    [[UIApplication sharedApplication] handleKeyUIEvent:keyboardEvent];
+    auto [keyboardEventDown, keyboardEventUp] = [capsLockOn = m_capsLockOn] {
+        auto *keyboardEventDown = createUIPhysicalKeyboardEvent(@"capsLock", [NSString string], capsLockOn ? UIKeyModifierAlphaShift : 0, kUIKeyboardInputModifierFlagsChanged, IsKeyDown::Yes);
```

Comment:
Nit - `[NSString string]` can just be `@""`

Reply:
Fixed